src/Templates/src/templates/maui-mobile/MauiApp.1.csproj

```diff
@@ -15,8 +15,10 @@
 
 		<!-- App Identifier -->
 		<ApplicationId>com.companyname.mauiapp._1</ApplicationId>
+		<ApplicationId Condition="$(TargetFramework.Contains('-windows'))">919dc1f9-17a9-48b3-81f8-0b8016bdfbf7</ApplicationId>
```

Comment:
If you put this in a template, we need to call `Guid.NewGuid()` in the template somehow.

Otherwise, everyone will have the same app id.

Reply:
The template engine has a feature to replace guids if you add them to a property: https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-mobile/.template.config/template.json#L53

Reply:
@mattleibow - Each GUID in the template that needs to be replaced upon creation also needs to be listed in the `template.json` file that you linked. Does this PR need to update template.json?

Reply:
Oh sorry nevermind I see this one is listed. Ignore!